### DISCUSSION TITLE
Template summary with <<collection_name>> variable?

### DISCUSSION THREAD

**Discussion by bigdork** [meisnate12/Plex-Meta-Manager#224] [Q&A] [answered]
Is this possible? For instance for Genre collections, I'd want to include the collection name in the summary instead of defining a unique summary for every collection. 

`   Summary: A collection of movies from the <<collection_name>> genre.`

Currently, this places the text "<<collection_name>>" in the summary instead of the actual collection name, ie: "Comedy"

**Comment by meisnate12** [comment, marked as answer]
are you using a template that should work with a template

**Reply by bigdork** [reply]
Well crap. Just after posting I noticed a typo in my config (missing > character). Disregard!

**Reply by bigdork** [reply]
Since I have you, possible to insert a carriage return/new line in summary?

**Reply by meisnate12** [reply]
discord is the best way to reach me fast hop on some time and if you put the string in quotes you should be able to use \n or \r `"Some Test \n Second line text"`
